vaadin-spreadsheet-flow-parent/vaadin-spreadsheet-flow-integration-tests/src/test/java/com/vaadin/flow/component/spreadsheet/test/CustomEditorIT.java

```diff
@@ -124,9 +124,9 @@ public void setValueToTextAreaCell_setFormulaInAnotherCell_valueAppliedToToCell(
 
         int sampleNumber = 42;
         setEditorValue("E2", Integer.toString(sampleNumber), "textarea");
-        setCellValue("E3", "=E2*2");
+        setCellValue("E4", "=E2*2");
         Assert.assertEquals(Integer.toString(sampleNumber * 2),
-                getCellValue("E3"));
+                getCellValue("E4"));
```

Comment:
This was not part of the original PR, but it's a change that has been done at [the feature PR](https://github.com/vaadin/flow-components/pull/7435/files#diff-f3b627a1cd65fccfc61ea602dbba44400d01d049e124b702d19660e068ecd7a3). For some reason, the default input doesn't receive the commands well after the first part of the test is performed. It's not reproducible when testing the same behavior manually. Changing the cell for the second part of the test makes the test more reliable.